botan-src/Cargo.toml

```diff
@@ -0,0 +1,23 @@
+[package]
+name = "botan-src"
+version = "0.1.0+2.15.0"
```

Comment:
I'd rather the crate version make the underlying version visible by packing the major minor and patch versions of Botan into the crate's minor version eg here "0.21500.0" where we assume the Botan minor and patch versions will never be > 99, and then the final patch version is reserved for the crate itself.

Maybe there is some problem with this, I haven't done any vendoring of sources in this manner in Rust. But having the crate match "0.1" regardless of what the underlying library is pinned to seems bad to me also.